test/5TRB.tml

```diff
@@ -0,0 +1,13 @@
+=== Invalid document-start marker in doublequoted tring
+from: 
+tags: 
+
++++ in-yaml
+--- "
```

Comment:
To make this test case YAML 1.3 compatible, the scalar should start at a new line (we forbade scalar content directly after `---` in 1.3).